src/components/sbb-form-field/sbb-form-field.stories.tsx

```diff
@@ -138,23 +138,10 @@ const TemplateInputWithIcons = (args): JSX.Element => (
   </sbb-form-field>
 );
 
-const TemplateInputWithButton = ({ disabled, readonly, active, ...args }): JSX.Element => (
+const TemplateInputWithClearButton = ({ disabled, readonly, ...args }): JSX.Element => (
   <sbb-form-field {...args}>
     {TemplateBasicInput({ ...args, disabled, readonly })}
-    <sbb-button
-      slot="suffix"
-      icon-name="cross-small"
-      disabled={disabled || readonly}
-      aria-label="clear input"
-      data-active={active}
-      onClick={() => {
-        const input = document.querySelector('input');
-        if (input) {
-          input.value = '';
-          input.focus();
-        }
-      }}
-    />
+    <sbb-form-field-clear />
```

Comment:
Can you still provide an additional example with an sbb-button (including all states as before), as we need to cover this case also with chromatic? Can be without any coupled logic, just for styles.